### ISSUE TITLE
Incorrect format type for barActiveThumbColor attribute

### ISSUE THREAD

**Issue by stephen-rival** [stfalcon-studio/StfalconPriceRangeBar-android#7]
The attribute for barActiveThumbColor is defined as:
`<attr name="barActiveThumbColor" format="dimension" />`

When it is read in by the View, it is done so like this:
```
 thumbActiveColor = typedArray.getColor(
            R.styleable.PriceRangeBar_barActiveThumbColor,
            thumbActiveColor
        )
```

I believe the attribute should be defined as:
`<attr name="barActiveThumbColor" format="color" />`

**Comment by andriizhumela** [comment]
Hi @stephen-rival !
You're right. It will be fixed in next version.

**Comment by andriizhumela** [comment]
@stephen-rival fixed in version 0.1.3
